elFinder.NetCore/Drivers/FileSystem/FileSystemFile.cs

```diff
@@ -8,22 +8,17 @@ namespace elFinder.NetCore.Drivers.FileSystem
 {
     public class FileSystemFile : IFile
     {
-        private readonly string filePath;
+        private FileInfo fileInfo;
```

Comment:
- Use FileInfo object to reuse and ensure an operation (unit of work) uses the same states of file.
- Gain performance by bypass File... checking. See: [FileInfo Remarks](https://docs.microsoft.com/en-us/dotnet/api/system.io.fileinfo?view=net-5.0#remarks)